shared/chat/conversation/index-hoc.js

```diff
@@ -106,18 +95,7 @@ const propsHoc = withProps(
       selectedConversation,
     }
 
-    const headerProps: HeaderProps = {
```

Comment:
I don't understand why you had to get rid of this:

This is what I mean: https://github.com/keybase/client/commit/16ceed8ecdd2a845a395807283442df976ab29d9

Then you don't need to be copying the same changes across desktop/native. You could control the props passed in once here.